### ISSUE TITLE
Automatic font scale according to accessibility option "font size" of device does not work on Android

### ISSUE THREAD

**Issue by ghost** [pichillilorenzo/flutter_inappwebview#540]
- [x] I have read the [README](https://github.com/pichillilorenzo/flutter_inappwebview/blob/master/README.md)
- [x] I have already searched for the same problem
- [x] I have read and done the setup for Android: [IMPORTANT Note for Android and iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android-and-ios) and [IMPORTANT Note for Android](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android)
- [x] I have read and done the setup for iOS: [IMPORTANT Note for Android and iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android-and-ios) and [IMPORTANT Note for iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-ios)
- [x] I have ran the sample app and it does not work there

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |     1.20.4          |
| Plugin version       |     4.0.0+4          |
| Android version      |       10        |
| iOS version          |       14        |
| Xcode version        |               |

Device information: Nokia 5.3 and iPhone 6s Plus

## Description

Users with eye sight impairments have the option to generally use a bigger font on their devices using the accessibility settings of the device. On Android the webview should automatically respect the accessibility settings and scale fonts accordingly when Javascript is enabled (https://a11y-guidelines.orange.com/mobile_EN/dev-android.html). 

The InAppWebView does not scale fonts according to the accessibility settings (unless I am missing something here!). 

On iOS scaling needed to be enabled through the CSS selector `font: -apple-system-body;` and on this platform it works as expected with the InAppWebView.

**Expected behavior:** 

If the user choses a larger font size in the accessibility settings of his Android device, the webview should scale the fonts  accordingly.

**Current behavior:** 

On Android the InAppWebView does not respect the accessibility settings.

**Comment by ghowen** [comment]
Additional info: It looks like the InAppWebView does not respect the font size settings, but it does respect the display size setting of the accessibility settings. Chrome on the other hand respects both. So I still suspect that the InAppWebView could respect that too.

**Comment by uc-dve** [comment]
Any update on it, it is working fine on webview_flutter?

**Comment by PawelSzymanski89** [comment]
any up?

**Comment by PawelSzymanski89** [comment]
@ghowen i have workaround
add this:
` android: AndroidInAppWebViewOptions(
            textZoom: (MediaQuery.of(context).textScaleFactor * 100).ceil(),`

in init options

**Comment by Den-creator** [comment]
What about iOS ? How to scale font size for the whole page ?
I see related not answered question - https://github.com/pichillilorenzo/flutter_inappwebview/issues/69.

**Comment by pichillilorenzo** [comment]
Released new version `6.2.0-beta.2` with the fix.
